### PR TITLE
Refactor/namespace data

### PR DESCRIPTION
**Pull request by m4rtint**

### CODE REVIEW

File: Assets/_PartyParkade/Script/Data/CarObject.cs [hunk 1]
```diff
@@ -11,34 +11,38 @@
...
-/// </summary>
-public struct CarObject
+namespace _PartyParkade.Script.Data
 {
-    private readonly Material m_Material;
-    private readonly Color m_Color;
-
     /// <summary>
-    /// Player Color
+    /// Struct CarObj
```
Comment (mikeslgoh) [inline Assets/_PartyParkade/Script/Data/CarObject.cs]: Learning more about C#. Any difference with doing this?

Comment (m4rtint) [reply Assets/_PartyParkade/Script/Data/CarObject.cs]: Do what? Create it as a struct?

Comment (m4rtint) [reply Assets/_PartyParkade/Script/Data/CarObject.cs]: Simple dumb object for data only. 
no logic, so struct

Comment (mikeslgoh) [reply Assets/_PartyParkade/Script/Data/CarObject.cs]: Yeah was just wondering why use a struct

File: Assets/_PartyParkade/Script/Data/Utilities.cs [hunk 2]
```diff
@@ -3,19 +3,19 @@
...
-    using System.Collections.Generic;
-    using System.Linq;
-    using InControl;
-    using UnityEngine;
 
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace _PartyParkade.Script.Data
```
Comment (wbeltran777) [inline Assets/_PartyParkade/Script/Data/Utilities.cs]: What parts of our code qualify as being "data"? Is it stuff that's accessed before the first frame of gameplay?

Comment (m4rtint) [reply Assets/_PartyParkade/Script/Data/Utilities.cs]: More like stuff that is read on run time. Static stuff that never changes.

Comment (m4rtint) [reply Assets/_PartyParkade/Script/Data/Utilities.cs]: Oh and it's just the name of the folders

**Comment by m4rtint** [comment]
[Add namespaces for Scripts in Data Folder](https://trello.com/c/Vi9Slj8Y/434-add-namespaces-for-scripts-in-data-folder)

**Pull request merged**